Price updates

deadlight opened the pull request:
## Done

Updated the price information of various cloud products.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/cloud/plans-and-pricing](http://0.0.0.0:8001/cloud/plans-and-pricing)
- See that the updates in the [copydoc](https://docs.google.com/document/d/1yZ7wClcmxGCqMyyyBJLO66Z3YlpT-4soWLFmJ6uXeMM/edit) are all addressed
- Go to: [http://0.0.0.0:8001/support/plans-and-pricing](http://0.0.0.0:8001/support/plans-and-pricing)
- See that the included shared portions updated for the other page look good here too


## Issue / Card

Fixes #2077

webteam-app commented:
Starting demo at: http://www.ubuntu.com-pr-2095.run.demo.haus//

joanasa89 commented:
Looks ok to me, just waiting for Frankie's final review to change my label.

anthonydillon commented on templates/cloud/plans-and-pricing.html:
```diff
@@ -13,24 +13,55 @@
+          <tr>
+            <th scope="row"><a href="/cloud/juju">Juju</a></th>
+            <td>FREE</td>
+            <td>Find out more at jujucharms.com </td>
```
Make this an external link please.

anthonydillon commented on templates/cloud/plans-and-pricing.html:
```diff
@@ -13,24 +13,55 @@
+            </td>
+            <td>
+              SaaS included as part of Ubuntu Advantage<br />
+              $2,000 per year - Landscape on-premises<br />
```
Encode the dash please.

anthonydillon commented on templates/shared/_ubuntu_advantage_cloud_per_node.html:
```diff
@@ -1,30 +1,30 @@
+      <h3 class="p-heading--five">Ubuntu Advantage support for OpenStack</h3>
+      <p class="p-heading--two">$1,500 <span class="p-heading--six">per year</span></p>
+    </div>
+    <p class="p-card__content"></p>
```
Can you remove this  empty element please

anthonydillon commented on templates/shared/_ubuntu_advantage_cloud_per_node.html:
```diff
@@ -1,30 +1,30 @@
+      <h3 class="p-heading--five">BootStack &mdash; Fully managed OpenStack</h3>
+      <p class="p-heading--two">$15 <span class="p-heading--six">a day per node</span></p>
+    </div>
+    <p class="p-card__content"></p>
```
Remove empty element please

deadlight commented on static/sass/styles.scss:
```diff
@@ -313,3 +313,9 @@ sup {
+
+/// XXX Give card headers an underlined option
+/// This will need replacing when the latest vanilla card changes land
+.p-card__header--bordered {
```
I think that I'll clean this up better. The problem is existing card headers on the site not having borders.

anthonydillon commented:
QA issues:
- The intro text is in one paragraph in the copy doc.
- The link to Landscape in the cloud tools section should be to: /server/management you have /support
- In the Service provider section there is a Buy Ubuntu Advantage link which is not in the copy doc
- I think the "BootStack fully managed OpenStack Cloud" section should be in a `col-9` to stop the ophan cloud in the title
- The link `Schedule a BootStack demo` should be to /cloud/contact-us?product=managed-cloud not /cloud/contact-us?product=cloud-plans-and-pricing
- The logo section in the "Ubuntu on public clouds" section does not match the copy doc

webteam-app commented:
Starting demo at: http://www.ubuntu.com-pr-2095.run.demo.haus//

webteam-app commented:
Starting demo at: http://www.ubuntu.com-pr-2095.run.demo.haus//

deadlight commented:
>The intro text is in one paragraph in the copy doc.
Updated the copydoc.

>The link to Landscape in the cloud tools section should be to: /server/management you have /support
Updated

>In the Service provider section there is a Buy Ubuntu Advantage link which is not in the copy doc
Added to copydoc

>I think the "BootStack fully managed OpenStack Cloud" section should be in a col-9 to stop the ophan cloud in the title
Done

>The link Schedule a BootStack demo should be to /cloud/contact-us?product=managed-cloud not /cloud/contact-us?product=cloud-plans-and-pricing
Updated in code and copydoc

>The logo section in the "Ubuntu on public clouds" section does not match the copy doc
The one in the code is correct. I'll update one of the two copydocs that use it and point the other one to it, to avoid duplicates getting out of sync.

deadlight commented:
@anthonydillon ready for re-review

@frankiegranato ready for UX review

anthonydillon commented:
Thanks @deadlight

deadlight commented:
@frankiegranato we're going live with this but please do a UX review when you're back.
